sound/creatures/license.txt

```diff
@@ -0,0 +1,2 @@
+dog_yelp.ogg is DogYelp.wav, by TobiasKosmos (https://freesound.org/people/TobiasKosmos/sounds/163280/). It has been licensed under CC-BY 3.0 license.
+It has been cropped, EQ'd, and compressed to .ogg for in-game use.
```

Comment:
Let's not add an entire text file fo a single sound.

Reply:
Where should I store license information?